pkgs/development/python-modules/sseclient/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, buildPythonPackage, fetchPypi
+, requests, six
+, backports_unittest-mock, pluggy, pytest, pytestrunner }:
+
+buildPythonPackage rec {
+  pname = "sseclient";
+  version = "0.0.18";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1n9f3rqrkzig27v0iki2flf56z4g4pwych40j1d82pf3p0bsyq0p";
+  };
+
+  propagatedBuildInputs = [ requests six ];
+
+  # it doesn't find any tests to run and then errors out
+  doCheck = false;
```

Comment:
We should probably fetch from GitHub in that case. I've opened an upstream PR: https://github.com/btubbs/sseclient/pull/12

Reply:
OK with things this way?

Reply:
Update is already available :)

Reply:
I have updated it to 0.0.19 and enabled the tests.